README.md

```diff
@@ -9,6 +9,46 @@ Ballerina SerDes Library
 
 This library provides APIs for serializing and deserializing subtypes of Ballerina anydata type.
 
+### Proto3Schema
+
+An instance of the `serdes:Proto3Schema` class is used to serialize and deserialize ballerina values using protocol buffers.
+
+#### Create a `serdes:Proto3Schema` object
+
+```ballerina
+// Define a type which is a subtype of anydata.
+type Student record {
+    int id;
+    string name;
+    decimal fees;
+};
+
+// Create a serialization object by passing the typedesc.
+serdes:Proto3Schema schema = check new (Student);
+```
+While instantiation of this object, an underlying proto3 schema generated for the provided typedesc.
+
+#### Serialization
+
+```ballerina
+Student studentValue = {
+    id: 7894,
+    name: "Liam",
+    fees: 24999.99
+};
+
+// Serialize the record value to bytes.
+byte[] serializedValue = check schema.serialize(studentValue);
```

Comment:
We usually don't define variable names like `studentValue`, right?

(You can come up with a better name than `bytes`.)

```suggestion
Student student = {
    id: 7894,
    name: "Liam",
    fees: 24999.99
};

// Serialize the record value to bytes.
byte[] bytes = check schema.serialize(student);
```

Reply:
Updated, sticking to `bytes` for now